### PR TITLE
Level 12 images

### PR DESCRIPTION
**Pull request by vEnhance**
still WIP, going to add the other images later

### CODE REVIEW

File: image-generator/yml/level-12/bluff-color-disconnect.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      - type: r
+        below:
+          text:
+            - Red 2
+            - Red 3
+          color: red
+      - type: x
+      - type: x
+      - type: x
+
```
Comment (Zamiell) [inline image-generator/yml/level-12/bluff-color-disconnect.yml:46]: the extra blank line is a bug

Comment (Zamiell) [reply image-generator/yml/level-12/bluff-color-disconnect.yml:46]: fix all of these

Comment (vEnhance) [reply image-generator/yml/level-12/bluff-color-disconnect.yml]: Done.

**Comment by Zamiell** [comment]
nice thanks

**Pull request merged**